### PR TITLE
grafana 4.1.2

### PR DESCRIPTION
**Pull request by bergquist**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by JCount** [comment]
This currently seems to cause node to hang when executing the `system "node_modules/yarn/bin/yarn", "install"` step. It hung on the CI and I was able to reproduce it locally I believe. Here is the error:
```
[4/4] Building fresh packages...
warning Error running install script for optional dependency: "/private/tmp/grafana-20170213-28407-v3i4hf/grafana-4.1.2/src/github.com/grafana/grafana/node_modules/fsevents: Cannot find module '../resolvers/index.js'"
info This module is OPTIONAL, you can safely ignore this error
verbose Error: /private/tmp/grafana-20170213-28407-v3i4hf/grafana-4.1.2/src/github.com/grafana/grafana/node_modules/node-sass: Cannot find module '../resolvers/index.js'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at _load_index (/private/tmp/grafana-20170213-28407-v3i4hf/grafana-4.1.2/src/github.com/grafana/grafana/node_modules/yarn/lib/util/execute-lifecycle-script.js:233:19)
    at /private/tmp/grafana-20170213-28407-v3i4hf/grafana-4.1.2/src/github.com/grafana/grafana/node_modules/yarn/lib/util/execute-lifecycle-script.js:124:51
    at Generator.next (<anonymous>)
    at step (/private/tmp/grafana-20170213-28407-v3i4hf/grafana-4.1.2/src/github.com/grafana/grafana/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /private/tmp/grafana-20170213-28407-v3i4hf/grafana-4.1.2/src/github.com/grafana/grafana/node_modules/babel-runtime/helpers/asyncToGenerator.js:28:13
```
This is complete overkill but [here](https://gist.github.com/JCount/bdcc4bd77021492d55d4f0395f9fff7e) is a sample of node when it hung.

**Comment by ilovezfs** [comment]
@bergquist I've gotten around the `yarn` issue by using `Language::Node.std_npm_install_args` which passes `--global` and installing yarn into `buildpath/"vendor/yarn"`. However, locally both @JCount and I are seeing

```
Running "karma:test" (karma) task
14 02 2017 11:41:07.220:INFO [karma]: Karma v1.3.0 server started at http://localhost:9876/
14 02 2017 11:41:07.221:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
14 02 2017 11:41:07.226:INFO [launcher]: Starting browser PhantomJS
14 02 2017 11:41:07.423:ERROR [phantomjs.launcher]: 2017-02-14 11:41:07.422 phantomjs[1146:2147689] -[_NSXPCDistantObjectWithError performScanWithChannels:ssidList:legacyScanSSID:includeHiddenNetworks:mergeScanResults:maxAge:maxMissCount:maxWakeCount:maxAutoJoinCount:interfaceName:waitForWiFi:waitForBluetooth:token:priority:reply:]: unrecognized selector sent to instance 0x7fdefe33f9d0

14 02 2017 11:41:07.423:ERROR [phantomjs.launcher]: PhantomJS has crashed. Please read the bug reporting guide at
<http://phantomjs.org/bug-reporting.html> and file a bug report.

14 02 2017 11:41:07.424:ERROR [launcher]: Cannot start PhantomJS
	
14 02 2017 11:41:07.427:INFO [launcher]: Trying to start PhantomJS again (1/2).
14 02 2017 11:41:07.633:ERROR [phantomjs.launcher]: 2017-02-14 11:41:07.633 phantomjs[1148:2147954] -[_NSXPCDistantObjectWithError performScanWithChannels:ssidList:legacyScanSSID:includeHiddenNetworks:mergeScanResults:maxAge:maxMissCount:maxWakeCount:maxAutoJoinCount:interfaceName:waitForWiFi:waitForBluetooth:token:priority:reply:]: unrecognized selector sent to instance 0x7f892a8375f0

14 02 2017 11:41:07.634:ERROR [phantomjs.launcher]: PhantomJS has crashed. Please read the bug reporting guide at
<http://phantomjs.org/bug-reporting.html> and file a bug report.

14 02 2017 11:41:07.634:ERROR [launcher]: Cannot start PhantomJS
	
14 02 2017 11:41:07.635:INFO [launcher]: Trying to start PhantomJS again (2/2).
14 02 2017 11:41:07.838:ERROR [phantomjs.launcher]: 2017-02-14 11:41:07.838 phantomjs[1149:2148006] -[_NSXPCDistantObjectWithError performScanWithChannels:ssidList:legacyScanSSID:includeHiddenNetworks:mergeScanResults:maxAge:maxMissCount:maxWakeCount:maxAutoJoinCount:interfaceName:waitForWiFi:waitForBluetooth:token:priority:reply:]: unrecognized selector sent to instance 0x7f91815507c0

14 02 2017 11:41:07.838:ERROR [phantomjs.launcher]: PhantomJS has crashed. Please read the bug reporting guide at
<http://phantomjs.org/bug-reporting.html> and file a bug report.

14 02 2017 11:41:07.839:ERROR [launcher]: Cannot start PhantomJS
	
14 02 2017 11:41:07.840:ERROR [launcher]: PhantomJS failed 2 times (cannot start). Giving up.
Warning: Task "karma:test" failed. Used --force, continuing.
```
unless `--force` is passed. I'd prefer not to ship this with `--force` in there, but it seems to fail on most actual hardware without it.

File: Formula/grafana.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Grafana < Formula
   desc "Gorgeous metric visualizations and dashboards for timeseries databases."
   homepage "http://grafana.org"
```
Comment (vszakats) [inline Formula/grafana.rb]: Please update this line to `homepage "https://grafana.org/"`

**Comment by torkelo** [comment]
please close this as there is now a newer version: https://github.com/Homebrew/homebrew-core/pull/11467

**Comment by JCount** [comment]
closing in favor of https://github.com/Homebrew/homebrew-core/pull/11467

**Pull request closed without merging**